.github/workflows/label-trigger.yml

```
@@ -0,0 +1,23 @@
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  label_actions:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)